types/leaflet-fullscreen/index.d.ts

```diff
@@ -10,4 +11,8 @@ declare module 'leaflet' {
     interface MapOptions {
         fullscreenControl?: true | {pseudoFullscreen: boolean} | undefined;
     }
+    interface Map {
+        isFullscreen: () => boolean;
+        toggleFullscreen: () => void;
```

Comment:
```ts
        toggleFullscreen: () => void;
```
Should this accept optional options?:
https://github.com/Leaflet/Leaflet.fullscreen/blob/gh-pages/dist/Leaflet.fullscreen.js#L59

Reply:
Yes. Just added the a type for the options and made it an optional arg to to `toggleFullscreen()`